nav2_behavior_tree/test/CMakeLists.txt

```diff
@@ -7,8 +7,3 @@ add_subdirectory(plugins/condition)
 add_subdirectory(plugins/decorator)
 add_subdirectory(plugins/control)
 add_subdirectory(plugins/action)
-
-install(DIRECTORY utils
-        DESTINATION include/)
-
-ament_export_include_directories(utils)
```

Comment:
I think you want to keep this line.

Reply:
It's handled by https://github.com/ros-planning/navigation2/pull/3692/files#diff-e551fb7ce40cf96abc6823ed49090bad21215d636841ab20a16e443b3c7b0380L235 already because it goes into the include folder.